build/root/Makefile.generated_files

```diff
@@ -449,7 +541,8 @@ $(foreach dir, $(OPENAPI_DIRS), $(eval             \
 # The Make rule fails if generated API rule violation report differs from the checked-in
 # violation file, and prints error message to request developer to fix either the API
 # source code, or the known API rule violation file.
-$(OPENAPI_OUTFILE): $(OPENAPI_GEN) $(KNOWN_VIOLATION_FILENAME)
+$(OPENAPI_OUTFILE): $(OPENAPI_GEN) $(OPENAPI_DIRS_FILE) $(KNOWN_VIOLATION_FILENAME)
+	echo $(OPENAPI_DIRS) > $(META_DIR)/openapi-dirs;                                \
```

Comment:
should this be `$(OPENAPI_DIRS_FILE)` instead of `$(META_DIR)/openapi-dirs`?

(also, the variable uses `openapi.dirs` rather than a -)